types/node/v9/index.d.ts

```diff
@@ -5659,6 +5660,38 @@ declare module "util" {
     export namespace promisify {
         const custom: symbol;
     }
+    // TypeScript Version: 2.1
```

Comment:
What is the use of this comment?

Reply:
I don't kown about it.if not has it,will get a error。